src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -3587,6 +3621,20 @@ type Stoptime {
     """The stop where this arrival/departure happens"""
     stop: Stop
 
+    """
+    The sequence of the stop in the pattern. This is not required to start from 0 or be consecutive - any
+    increasing integer sequence along the stops is valid.
+
+    The purpose of this field is to identify the stop within the pattern so it can be cross-referenced
+    between it and the itinerary. It is safe to cross-reference when done quickly, i.e. within seconds.
+    However, it should be noted that realtime updates can change the values, so don't store it for
+    longer amounts of time.
+
+    Depending on the source data, this might not be the GTFS `stop_sequence` but another value, perhaps
+    even generated.
+    """
+    stopPosition: StopPosition
```

Comment:
I wonder if we actually ever would return StopTime from an itinerary. Do we need to support the  PositionBetweenStops here or would be enough to have this as Int or PositionAtStop?

Reply:
That's actually a very good point - since the stoptime is associated with a single stop only, we can return an int here. I will fix this.